### PR TITLE
empty properties treated as null

### PR DESCRIPTION
**Pull request by jawitcher**

### CODE REVIEW

**Comment by jawitcher** [comment]
> 
> 
> @jawitcher Thanks. I had a look at the PR and it looks like what we need.
> 
> Could you also do the following and then change it from DRAFT to the PR?
> 
>     1. Add the same logic for `getMode()`
> 
>     2. Add unit tests
> 
>     3. Format the code
> 
> 
> Thanks!

I revised the code in accordance with your notes

File: src/test/java/com/hazelcast/kubernetes/KubernetesConfigTest.java [hunk 1]
```diff
@@ -275,4 +275,32 @@ private static String createTestFile(String expectedContents)
...
         return temp.getAbsolutePath();
     }
+
+    @Test
+    public void propertyServiceNameIsEmpty() {
+        // given
+        Map<String, Comparable> properties = createProperties();
+        properties.put(SERVICE_NAME.key(), "  ");
+        properties.put(SERVICE_DNS.key(), "service-dns");
+        new KubernetesConfig(properties);
```
Comment (leszko) [inline src/test/java/com/hazelcast/kubernetes/KubernetesConfigTest.java:285]:
```suggestion
        // given
        Map<String, Comparable> properties = createProperties();
        properties.put(SERVICE_NAME.key(), "  ");
        properties.put(SERVICE_DNS.key(), "service-dns");

        // when
        KubernetesConfig config = new KubernetesConfig(properties);

        // then
        assertEquals("service-dns", config.getServiceDns());
```

Could you change this test to this format? And could you also change the other tests the same way?

**Comment by mesutcelik** [comment]
@jawitcher Can you please sign the CLA before the merge?

**Comment by jawitcher** [comment]
@leszko , I changed it

**Comment by jawitcher** [comment]
> 
> 
> @jawitcher Can you please sign the CLA before the merge?

Hi. I signed it :). My name is on the list https://hazelcast.atlassian.net/wiki/spaces/COM/pages/6357071/Hazelcast+Contributor+Agreement

**Comment by jawitcher** [comment]
> 
> 
> @jawitcher Can you please sign the CLA before the merge?

Hi. I signed it :)

**Comment by leszko** [comment]
Looks great 💯 

Thanks a lot for the change @jawitcher !

**Pull request merged**